microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -190,15 +131,98 @@ You can access Collections from an InPrivate window when using a signed-out prof
 <!-- ====================================================================== -->
 ## Compose
 
-In the Copilot pane, the Compose tool uses AI to write for you, starting from a short prompt and **Tone**, **Format**, and **Length** selections. Text that's generated by Compose can be copied or inserted into the webpage directly.
+The Compose tool helps you draft text within the Microsoft Edge Sidebar, or rewrite selected text on any webpage.
+
+
+<!-- ------------------------------ -->
+#### Use Compose to draft text within the Edge Sidebar
+
+In the Copilot pane, the Compose tool uses AI to write for you, starting from a short prompt and **Tone**, **Format**, and **Length** selections:
```

Comment:
```suggestion
In the Copilot pane, the Compose tool uses AI to write for you, starting from a short prompt and **Tone**, **Format**, and **Length** selections.
```

Reply:
a minor change that could go either way.  check rendered view re: flow.  often in this repo, we use colon to glue the lead-in sentence to the subsequent png that that sentence leads into or introduces.

Reply:
subseq comment removes png, so no lead-in colon